### PR TITLE
Feature/totalizer responsiveness

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Add `mobileScroll` prop.

#### What problem is this solving?
Horizontal responsiveness to spare space on mobile.

#### How should this be manually tested?
checkout & `yarn styleguide`.

#### Screenshots or example usage
Screen Shot 2019-12-11 at 17 37 50

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Totalizer/index.js [hunk 1]
```diff
@@ -7,45 +7,68 @@ import TotalizerIcon from './TotalizerIcon'
...
   render() {
-    const { items } = this.props
+    const { items, mobileScroll } = this.props
+
+    const OUTER_CONTAINER_CLASSES = mobileScroll ? 'overflow-x-hidden' : ''
+
+    let INNER_CONTAINER_CLASSES = `w-100 flex flex-row-ns ba br3 mb5 ${BORDER_COLOR} `
+    INNER_CONTAINER_CLASSES += mobileScroll
+      ? 'flex-row overflow-y-hidden overflow-x-scroll overflow-x-hidden-ns'
+      : 'flex-column'
```
Comment (matheusps) [inline react/components/Totalizer/index.js]: Maybe `classnames` can help you here.

File: react/components/Totalizer/index.js [hunk 2]
```diff
@@ -7,45 +7,68 @@ import TotalizerIcon from './TotalizerIcon'
...
+
+    let TOTALIZER_BASE_CLASSES = `flex flex-column pa4 ${BORDER_COLOR} `
+    TOTALIZER_BASE_CLASSES += mobileScroll ? 'flex-auto-ns' : 'flex-auto'
```
Comment (matheusps) [inline react/components/Totalizer/index.js]: also

File: react/components/Totalizer/index.js [hunk 3]
```diff
@@ -7,45 +7,68 @@ import TotalizerIcon from './TotalizerIcon'
...
+          {items.map((item, i) => {
+            const IS_NOT_LAST = items.length > 1 && i < items.length - 1
+            let EXTRA_BORDER = IS_NOT_LAST ? 'bb bb-0-ns br-ns ' : ''
+
+            if (mobileScroll) {
+              EXTRA_BORDER += 'bb-0 '
+            }
+            if (mobileScroll && IS_NOT_LAST) {
+              EXTRA_BORDER += 'br '
+            }
```
Comment (matheusps) [inline react/components/Totalizer/index.js]: same

File: react/components/Totalizer/index.js (hunk 1 above)
Comment (kevinch) [reply react/components/Totalizer/index.js]: Good point.

Comment (kevinch) [reply react/components/Totalizer/index.js]: Updated.

File: react/components/Totalizer/index.js (hunk 2 above)
Comment (kevinch) [reply react/components/Totalizer/index.js]: Updated.

File: react/components/Totalizer/index.js (hunk 3 above)
Comment (kevinch) [reply react/components/Totalizer/index.js]: Updated.

File: react/components/Totalizer/index.js [hunk 4]
```diff
@@ -1,51 +1,82 @@
...
-          return (
-            <div
-              className={`${TOTALIZER_CLASSES} ${EXTRA_BORDER}`}
-              key={item.label}>
-              {item.icon ? (
-                <div className="flex flex-row">
-                  <TotalizerIcon item={item} />
+            return (
+              <div
+                className={`${TOTALIZER_BASE_CLASSES} ${EXTRA_BORDER}`}
```
Comment (emersonlaurentino) [inline react/components/Totalizer/index.js]: why you do not use `classnames`?

Comment (kevinch) [reply react/components/Totalizer/index.js]: Updated.

**Pull request merged**